tests/conftest.py

```diff
@@ -33,7 +33,7 @@ def pytest_sessionfinish(session: pytest.Session):
         ln.UPath("s3://lamindb-test/.lamindb").rmdir()
     ln.setup.delete("lamindb-unit-tests", force=True)
     # shutil.rmtree("./outside_storage")
-    run("docker stop pgtest && docker rm pgtest", shell=True, stdout=DEVNULL)
+    run("docker stop pgtest && docker rm pgtest", shell=True, stdout=DEVNULL)  # noqa: S602
```

Comment:
Is S602 saying that `shell=True` is OK?

Reply:
Yes! https://docs.astral.sh/ruff/rules/subprocess-popen-with-shell-equals-true/

1. Since this is only a test, we're fine with having `shell=True` being used.
2. This uses `&&` which is the shell interprets so we need `shell=True` (excluding the possibility to rewrite - wasted effort here for a test)